sdk/nodejs/src/open-feature/DevCycleProvider.ts

```diff
@@ -248,12 +249,22 @@ export class DevCycleProvider implements Provider {
     private resultFromDVCVariable<T>(
         variable: DVCVariableInterface,
     ): ResolutionDetails<T> {
-        return {
+        const resolutionDetails: ResolutionDetails<T> = {
             value: variable.value as T,
-            reason: variable.isDefaulted
-                ? StandardResolutionReasons.DEFAULT
-                : StandardResolutionReasons.TARGETING_MATCH,
+            //TODO: once eval enabled from cloud bucketing, eval won't be null unless defaulted
```

Comment:
Cloud bucketing should already be updated right? like we already deployed the CF workers?